test/basictests.jl

```diff
@@ -570,29 +570,34 @@ end
     # These are tests for NewtonRaphson so we should set alpha_initial to be high so that we
     # converge quickly
 
-    function benchmark_nlsolve_oop(f, u0, p = 2.0; alpha_initial = 10.0)
+    function benchmark_nlsolve_oop(f, u0, p = 2.0;
+            alpha_initial = 10.0, update_alpha = switched_evolution_relaxation)
         prob = NonlinearProblem{false}(f, u0, p)
-        return solve(prob, PseudoTransient(; alpha_initial), abstol = 1e-9)
+        return solve(prob, PseudoTransient(; alpha_initial, update_alpha), abstol = 1e-9)
     end
 
     function benchmark_nlsolve_iip(f, u0, p = 2.0; linsolve, precs,
-            alpha_initial = 10.0)
+            alpha_initial = 10.0, update_alpha = switched_evolution_relaxation)
         prob = NonlinearProblem{true}(f, u0, p)
-        return solve(prob, PseudoTransient(; linsolve, precs, alpha_initial), abstol = 1e-9)
+        return solve(prob,
+            PseudoTransient(; linsolve, precs, alpha_initial, update_alpha),
+            abstol = 1e-9)
     end
 
-    @testset "PT: alpha_initial = 10.0 PT AD: $(ad)" for ad in (AutoFiniteDiff(),
-        AutoZygote())
+    @testset "update_alpha:$(update_alpha) alpha_initial = $(alpha_initial) AD: $(ad)" for (update_alpha, alpha_initial) in zip((switched_evolution_relaxation,),
+            (10.0,)),
```

Comment:
This is done when were are iterating over multiple possibilities.

Reply:
I was just preparing for the next pr when we add robust PT, but if you don't like it, I can fix it